server/services/prismic-content.js

```diff
@@ -0,0 +1,180 @@
+import type {Article} from '../model/article';
+import type {Picture} from '../model/picture';
+import Prismic from 'prismic-javascript';
+import {RichText, Date as PrismicDate} from 'prismic-dom';
+import {prismicApiV2} from './prismic-api';
+
+export async function getContent(id) {
+  const prismic = await prismicApiV2();
+  const fetchLinks = [
+    'people.name', 'people.image', 'people.twitterHandle',
+    'books.title', 'books.title', 'books.author', 'books.isbn', 'books.publisher', 'books.link', 'books.cover',
+    'series.name','series.description', 'series.color', 'series.commissionedLength'
+  ];
+  const articles = await prismic.query(Prismic.Predicates.at('document.id', id), {fetchLinks});
+  const prismicArticle = articles.total_results_size === 1 ? articles.results[0] : null;
+
+  // TODO : construct this not from strings
+  const url = `/articles/${id}`;
+
+  // TODO: Leave this in the flow of the body
+  const prismicStandfirst = prismicArticle.data.body.find(slice => slice.slice_type === 'standfirst');
+  const standfirst = prismicStandfirst && RichText.asText(prismicStandfirst.primary.text);
+
+  // TODO: Add this to the article body
+  // TODO: potentially get rid of this
+  const publishDate = PrismicDate(prismicArticle.data.publishDate || prismicArticle.first_publication_date);
+
+  // TODO:
+  const mainMedia = prismicArticle.data.body.filter(slice => slice.primary.weight === 'featured').map(slice => {
+    return prismicImageToPicture(slice.primary);
+  });
+
+  // TODO: Don't convert this into thumbnail
+  const promo = prismicArticle.data.promo.find(slice => slice.slice_type === 'embeddedImage').primary;
```

Comment:
Yea - there's loads of `null` pointer exceptions here.

A bit of context here - this is just so editors can start testing Prismic, so I can't say this is the best code I've written.

I'll 100% get it typed etc though once we're 100% commited to this being the system.